### PR TITLE
added getValue() support for GPS devices

### PR DESCRIPTION
**Pull request by dsbeach**
Proposed changed for issue #2758

### CODE REVIEW

File: radio/src/lua_api.cpp [hunk 1]
```diff
@@ -129,6 +129,29 @@ static int luaGetDateTime(lua_State *L)
   return 1;
 }
 
+static void luaPushLatLng(TelemetrySensor & telemetrySensor, TelemetryItem & telemetryItem)
+/* result is lua table containing members ["Lat"] and ["Lng"] as lua_Number (doubles) in decimal degrees */
+{
```
Comment (projectkk2glider) [inline radio/src/lua_api.cpp]: We use `Lon` abbreviation for Longitute, please rename it everywhere you used `Lng`.

File: radio/src/lua_api.cpp [hunk 2]
```diff
@@ -138,7 +161,9 @@ static void luaGetValueAndPush(int src)
     // telemetry values
     if (TELEMETRY_STREAMING() && telemetryItems[src].isAvailable()) {
       TelemetrySensor & telemetrySensor = g_model.telemetrySensors[src];
-      if (telemetrySensor.prec > 0)
+      if (telemetrySensor.unit == UNIT_GPS)
+		  luaPushLatLng(telemetrySensor, telemetryItems[src]);
```
Comment (projectkk2glider) [inline radio/src/lua_api.cpp]: Fix indentation, we use two spaces (no tabs) for each level.

**Comment by projectkk2glider** [comment]
Hi, please don't take my comments negative. We appreciate your work, but we also like to keep up the coding style. 

The pull itself looks good, thanks. Did you test it?

**Comment by dsbeach** [comment]
Your comments are appreciated and understood.  Had I been paying more attention I would have noticed the Lng->Lon and the tab key that I missed.

Regarding testing the answer is yes and no.  For some reason the git repository where I generated the build request is now building 2.0.99 and I'm not sure why.  I'm obviously not completely up to speed on this project and the tools.

**Comment by bsongis** [comment]
2.0.99 then it's old, perhaps you should cherry-pick your commit on a new
branch coming from 'master'

**Comment by kilrah** [comment]
> dsbeach:2758: This branch is 2 commits ahead, 1171 commits behind opentx:master

You forgot to pull the changes from our repo into yours before doing your work. Github doesn't keep forks up to date itself, so at this point your master branch is where ours was when you did your fork a few months ago.

You should add https://github.com/opentx/opentx.git as a second remote in your git client, then pull/rebase our master branch.

**Pull request closed without merging**